fixes.inc

```diff
@@ -6406,6 +6498,31 @@ native BAD_TogglePlayerControllable(playerid, toggle) = TogglePlayerControllable
 #endif
 
 /*
+ * FIXES_StopPlayerEnterVehicle(playerid, bool:kick)
+ *
+ * Block a player from entering a vehicle after the animation for a locked driver door
+ *
+ * FIXES:
+ *     OnPlayerEnterVehicle_3
+ */
+
+#if FIX_OnPlayerEnterVehicle_3
+	static stock FIXES_StopPlayerEnterVehicle(playerid)
```

Comment:
The function should be renamed to `_FIXES_StopPlayerEnterVehicle` ([guidelines](https://github.com/Open-GTO/sa-mp-fixes/blob/master/fixes.inc#L859-L861)).